azure-pipelines.yml

```diff
@@ -391,19 +391,6 @@ stages:
       - powershell: eng/build.ps1 -configuration Release -prepareMachine -ci -build -pack -publish -sign -binaryLogName Build.binlog /p:DotnetPublishUsingPipelines=true
         displayName: Build
 
-      - task: PowerShell@2
```

Comment:
This task is intentional as it validates our symbol publishing prior to merging.

Reply:
I don't think it's doing much of value, as this task is not executed on an official build.

Reply:
The official build does the actual publishing as part of V3 publishing so it does not need to run this task. This does a dry-run during PR CI to ensure our publishing and exclusion configuration is correct. When we moved this correctness task from V2 to V3 publishing in order to take advantage of exclusions, I did a run that verified this was catching issues.

Reply:
Sounds good.

Reply:
> The official build does the actual publishing as part of V3 publishing so it does not need to run this task. This does a dry-run during PR CI to ensure our publishing and exclusion configuration is correct. When we moved this correctness task from V2 to V3 publishing in order to take advantage of exclusions, I did a run that verified this was catching issues.

Would be great if this explanation was present as a comment in the code...